sdks/python/apache_beam/dataframe/io.py

```diff
@@ -0,0 +1,178 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+from io import BytesIO
+from io import StringIO
+from io import TextIOWrapper
+
+import pandas as pd
+
+import apache_beam as beam
+from apache_beam import io
+from apache_beam.dataframe import frame_base
+
+
+def read_csv(path, *args, **kwargs):
+  return _ReadFromPandas(pd.read_csv, path, args, kwargs)
+
```

Comment:
Is there a reason to provide these as methods instead of transforms (similar to other IO connectors) ?

Reply:
Mirroring the Pandas APIs. The _ReadFromPandas will be widely shared.